config/infra-sandbox/nextflow-vpc.yaml

```diff
@@ -0,0 +1,11 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.3.0/templates/VPC/vpc-20-private.yaml"
+stack_name: nextflow-vpc
+
+parameters:
+  VpcName: nextflow-vpc
+  VpcSubnetPrefix: "172.23"
+
+stack_tags:
+  {{stack_group_config.default_stack_tags}}
```

Comment:
Is there a reason you can't use sandcastle? Does it have to do with subnets perhaps?

Reply:
I considered it, but the Tower configuration script looks for a CFN stack called `nextflow-vpc` and currently assumes four private subnets (sandcastle has 3 public and 3 private). 

I figured that this VPC would eventually be re-deployed to the AMP-AD, so this PR can basically undergo `s/sandbox/ampad/g` and it should work.